src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+
+// ACTIONS
+export function addBook() {
+  return {
+    type: 'ADD_BOOK',
+  };
+}
+
+export function removeBook() {
```

Comment:
- [ ] Please, use ES6 best practices, in this case, this could improve being an arrow function. 🦊